src/components/elements/select.tsx

```diff
@@ -0,0 +1,2 @@
+"use client"
+import React from "react"
```

Comment:
I know we prefer to have components but then realized it made it harder to see where I broke it. Is it better to get it to work and then break out the components?